src/main/java/com/j256/ormlite/stmt/Where.java

```diff
@@ -568,6 +568,19 @@ public String toString() {
 			return "where clause: " + clause;
 		}
 	}
+	
```

Comment:
Can you move this up to be right after `getStatement()`.  `toString()` is at the bottom for me and the `appendSql(...)` isn't public.  Thanks.